src/config/dynamic-theme-fixes.config

```diff
@@ -2898,32 +2898,83 @@ span[class^="burger"]
 codeforces.com
 
 INVERT
-#header > div:first-child img
-.action-link > div > img:first-child
-.roundbox-lt
-.roundbox-rt
-.roundbox-lb
-.roundbox-rb
+#userActivityGraph rect[data-items=""]
 .delete-resource-link
-a.contestParticipantCountLinkMargin > img
-.recent-actions > ul > li img[src*="hourglass"]
+.action-link img
+.non-decorated img[src$="tablesorter-bg.gif"]
 
 CSS
+#header > div:first-child > a::before {
+    content: "";
+    display: inline-block;
+    height: 65px;
+    width: 15px;
+    border-top-left-radius: 12px;
+    border-bottom-left-radius: 12px;
+    background-color: #fff;
+}
+#header > div:first-child > a img {
+    border-radius: 0 12px 12px 0;
+}
+#header > div:first-child img {
+    border-radius: 12px;
+}
+.datatable,
+.datatable > div,
+.roundbox {
+    border-radius: 6px;
+}
 .lt,
 .rt,
 .lb,
 .rb,
 .ilt,
-.irt {
+.irt,
+.roundbox-lt,
+.roundbox-rt,
+.roundbox-lb,
+.roundbox-rb {
     display: none;
 }
-input[type="submit"], input[type="button"], select {
-    border-style: solid;
+.backLava,
+.leftLava {
+    border-radius: 4.5px;
+}
+.backLava,
+.leftLava,
+.bottomLava,
+.cornerLava {
+    background: ${#BEBEBE} !important;
+}
+input[type="submit"],
+input[type="button"],
+select {
+    border-width: 1px;
+    border-style: inset;
+    border-radius: 3.5px;
+}
+input[type="submit"]:hover,
+input[type="button"]:hover {
+    border-style: outset;
+}
+input[type="submit"]:hover,
+input[type="button"]:hover {
+    transform: translate(1px, 1px);
+}
+.sidebar-menu ul li {
+    border-color: transparent !important;
+}
```

Comment:
The white version has white (=invisible, since white on white) borders, so I made them invisible in black